client/styles/components/_filter-form.scss

```diff
@@ -1,30 +1,40 @@
 .filter-form {
-  @include mq(tablet) {
-    float: left;
-    width: 35%;
+  flex-basis: 34px;
+
+  @include mq(largeTablet) {
+    flex-basis: 330px;
+    margin: 0 20px;
   }
 }
 .filter-form__inner {
-  padding-left: 80px;
+  display: flex;
+  align-items: center;
 }
 .filter-form__label {
-  float: left;
-  margin-left: -80px;
-  padding-top: 10px;
+  margin-right: 10px;
 }
 .filter-form__select {
-  float: left;
+  background-color: white;
+  border-radius: 2px;
+  // float: left;
+  min-width: 205px;
```

Comment:
I think this is still a bit big - at 320px (a small mobile phone) it goes wider than the search bar. Maybe just set this to 100px - it will always be bigger than that anyway.